src/features.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0"?>
+<featureStaging xmlns="http://microsoft.com/TilFeatureStaging-Schema.xsd">
+    <!-- See doc/feature_flags.md for more info.  -->
+    <feature>
+        <name>Feature_ExampleFeat</name>
+        <description>This feature will be replaced in the next pull request.</description>

```

Comment:
This is https://github.com/microsoft/terminal/compare/dev/duhowett/jerk...dev/duhowett/jerkify

Reply:
LOL ok.